README.md

```diff
@@ -11,23 +11,32 @@ Interface to FreeType to generate font bitmaps run time
 - Add library: `freetype` (for GCC: `-L/usr/local/lib -lfreetype`)
 
 ## Usage in LVGL
+
+To enable cache, set`LV_USE_FT_CACHE_MANGER  1`in lv_freetype.h.
+
 ```c
-lv_freetype_init(64); /*Cache max 64 glyphs*/
+/* init freetype library */
+lv_freetype_init(64, 1, 0);
 
 /*Create a font*/
-static lv_font_t font1;
-lv_freetype_font_init(&font1, "./lv_lib_freetype/arial.ttf", 32);
+lv_ft_info_t info;
+info.name = "./lv_lib_freetype/simhei.ttf";
```

Comment:
Please leave it as Arial, as that is probably the more common font and simhei.ttf is not in the repository.